assets/src/edit-story/components/canvas/frameElement.js

```diff
@@ -49,13 +49,14 @@ const Wrapper = styled.div`
 	&:focus,
 	&:active,
 	&:hover {
-		outline: 1px solid ${({ theme }) => theme.colors.selection};
+		outline: ${({ theme, hasMask }) =>
```

Comment:
I don't think this is quite the same purpose. We need to show outline to clarify what can be moved - i.e. it's a `Moveable` feature. We need to keep it separate for dropping. E.g. if a text is cut off - we still want to show the full outline when hovering/focusing.

Reply:
Yes, this is a replacement only for maskeable elements (for text, we keep the same behavior). All it does is essentially adapt the outline to the mask's shape instead of having a rectangular selection box for masked elements. This is something Sam brought up, but again we can revisit in a design review.